exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/join/TestNestedLoopJoin.java

```diff
@@ -333,4 +341,74 @@ public void testNlJoinWithLargeRightInputSuccess() throws Exception {
       test(RESET_JOIN_OPTIMIZATION);
     }
   }
+
+  private void buildFile(String fileName, String[] data, File testDir) throws IOException {
+    try(PrintWriter out = new PrintWriter(new FileWriter(new File(testDir, fileName)))) {
+      for (String line : data) {
+        out.println(line);
+      }
+    }
+  }
+
+  @Test
+  public void testNestedLeftJoinWithEmptyTable() throws Exception {
+    setSessionOption("planner.enable_join_optimization", false);
```

Comment:
Generally a better idea is to use the constants defined in `ExecConstants`. Doing so avoids any possibility of introducing a typo in the option name.